CMakeLists.txt

```diff
@@ -28,7 +28,6 @@ proExecuteStep()
 
 if(${XP_STEP} STREQUAL "build")
 # move some needed cmake files into place
-configure_file(Findovsrpro.cmake ${STAGE_DIR}/share/cmake/Findovsrpro.cmake COPYONLY)
```

Comment:
Shouldn't this be 
```configure_file(/opt/extern/externpro/Findscript.cmake.in ${STAGE_DIR}/share/cmake/Findovsrpro.cmake @ONLY)```

Reply:
no - he's deleting this line

also - commenting on your suggestion: you would never want a hard-coded path like `/opt/extern` (the rest of the path isn't correct, either) - you want to "find" the installed externpro and use the script from share/cmake

I attempted to explain how the script is created in comments in this pull (outdated and hidden now) https://github.com/distributePro/ovsrpro/pull/30#discussion_r163688291

Reply:
Ahh, I understand now.   Thanks.

@smanders  What is the advantage of copying this file instead of adding the install path to directories searched by find_package?

Reply:
@tjflexmaster maybe I don't understand your question - are you asking if you could replace `Findscript.cmake.in` (which generates `Findovsrpro.cmake`) with `find_package`? this file (`Findscript.cmake.in`) does a lot more than `find_package` -- but I suppose you could replace it if you wanted to add the complications to the project that uses the *pro project...

Reply:
@smanders Let me try again.  If I call find_package(ovsrpro MODULE REQUIRED) in my cmake it can find and load the Findovsrpro.cmake that is installed when ovsrpro was installed (/opt/extern/ovsrpro/share/cmake/Findovsrpro.cmake).  All I have to do is add /opt/extern/ovsrpro... to the CMAKE_MODULE_PATH list.  I am not talking about replacing Findscript.cmake.in I am talking about using the file generated by the build directly instead of using a copy of it.

Reply:
@tjflexmaster you don't like the copy? is there something difficult about it? or just trying to simplify?

I believe your simplifying assumption is that you only look in `/opt/extern` -- but you also need the ovsrpro directory with all it's compiler, bits, and OS decorations -- the `Findscript.cmake.in` supports multiple locations to look for it: developer "build" versions, using an environment variable, and multiple install locations -- Windows-specific and UNIX-specific

would you suggest adding all of those locations to your `CMAKE_MODULE_PATH`?

you are correct -- if you only support a single location to find ovsrpro then you could skip the copy to your project that uses ovsrpro and add the install location to `CMAKE_MODULE_PATH` so `find_package()` could find it

Reply:
@smanders The copy isn't a big deal.  I was just wondering about the reasons.  I typically only have ovsrpro installed in one location so telling cmake what that is through a cmake variable at configuration time or through environment variables is easy enough.  But with the number of developers and the different number of environments you need to support I can see the benefit of taking this duty off of the developer.

Reply:
@tjflexmaster it's always good to ask the questions... sometimes it will make me rethink why I did do things the way I did -- so many of these things were done in my early days of cmake, too -- so I won't be the least bit surprised if there are better ways to do some of the things I've done -- but I have put a lot of thought and trial and error into what there is in externpro today (over the years)